notebook/notebookapp.py

```diff
@@ -608,6 +608,9 @@ def _webapp_settings_changed(self, name, old, new):
             help="Supply overrides for the tornado.web.Application that the "
                  "Jupyter notebook uses.")
     
+    terminado_settings = Dict(config=True,
+            help='Supply overrides for terminado. Currently only supports "shell_command".')
```

Comment:
I didn't want to add anything I didn't need right now. I could see someone wanting to add environment variables or , for instance. It also seems to me that it is also consistent with `tornado_settings`.